dashboard/src/actions/datasetListActions.js

```diff
@@ -36,6 +36,7 @@ export const fetchDatasets = (page) => async (dispatch, getState) => {
       getState().datasetlist;
     const datasetType = getState().comparison.datasetType;
     let publicData = [...getState().datasetlist.publicData];
+
     const params = new URLSearchParams();
     addParams(params, loggedIn, datasetType);
     params.append("offset", offset);
```

Comment:
The only change to this module now is the addition of this one blank line.  It would have been nice to revert this change to minimize the churn, but let's just merge and move on.